lib/services/service_brokers/v2/client.rb

```diff
@@ -79,15 +79,35 @@ def fetch_service_instance_state(instance)
       result.merge(parsed_response.symbolize_keys)
     end
 
+    def create_service_key(key, arbitrary_parameters: {})
```

Comment:
Hi Tom,

It's a good point. I don't think we'll ever want to have both endpoints (one for create-service-key and another for service-binding), but I also think that your implementation might be more clear than the one I suggested. Let's stick with two methods for now, and if the duplicate code becomes a problem we can merge them later.

I'll merge this as soon as the pipeline goes green.